gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/CacheServerLauncher.java

```diff
@@ -1546,7 +1546,13 @@ protected void waitForRunning(final Properties props) throws Exception {
           }
       }
       writePidToFile(status);
-      System.out.println( status );
+      if (status.state != WAITING) {
+        System.out.println(status);
+      } else {
+        System.out.println(status.shortStatus() + ": " +
```

Comment:
We should give a little more information that for which member it is waiting. In bracket we can give the diskstoreId as well. Only giving this much will make the user scramble to check all members which might be bad if it is a big cluster.